.travis.yml

```diff
@@ -11,5 +13,5 @@ before_script:
   - cp -R Tests/ vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/Symfony/Tests/
 
 script:
-  - (cd vendor/squizlabs/php_codesniffer ; phpunit)
-  - ./vendor/squizlabs/php_codesniffer/scripts/phpcs ./vendor/squizlabs/php_codesniffer/CodeSniffer/Standards/Symfony/Sniffs --standard=PEAR --report=summary -np
+  - (cd vendor/squizlabs/php_codesniffer ; phpunit --filter Symfony_)
```

Comment:
https://travis-ci.org/xalopp/symfony-coding-standard/jobs/53080394#L111 says "No tests executed!" is that expected?